test/views/HomePageViewSpec.scala

```diff
@@ -109,6 +110,10 @@ class HomePageViewSpec extends TestSupport with FeatureSwitching {
     LocalDate.of(2018, 2, 1), LocalDate.of(2018, 3, 1)), currentDate, false)
   private val viewModelNoUpdates: NextUpdatesTileViewModel = NextUpdatesTileViewModel(Seq(), currentDate, false)
   private val viewModelOptOut: NextUpdatesTileViewModel = NextUpdatesTileViewModel(Seq(LocalDate.of(2100, 1, 1)), currentDate, true)
+  val todayDateMinusOneDay: LocalDate = LocalDate.now().minusDays(1)
```

Comment:
Please don't use LocalDate.now anywhere in the tests, we rely on the fixed dates instead

Reply:
Will update the tests